### PR TITLE
Cut 0.25.2

### PR DESCRIPTION
**Pull request by r2d4**
After LGTM i'll cut the tag off of this PR and do a release.

### CODE REVIEW

**Comment by r2d4** [comment]
https://github.com/kubernetes/minikube/compare/v0.25.1...v0.25.2

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Minikube Release Notes
 
+# Version 0.25.1 - 3/22/2018
```
Comment (afbjorklund) [inline CHANGELOG.md]: 0.25.1 again ?

**Comment by afbjorklund** [comment]
Looks like the changelog had a copy/paste error (wrong version) ?

**Comment by afbjorklund** [comment]
@r2d4 : where is the git branch for 0.25, that this talks about ? I can only seem to find the git tags ?

https://github.com/kubernetes/minikube/releases/tag/v0.25.2
https://github.com/kubernetes/minikube/releases/tag/v0.25.1
https://github.com/kubernetes/minikube/releases/tag/v0.25.0

There is a https://github.com/kubernetes/minikube/tree/k8s-v1.9.4 but it doesn't seem to be merged ?

f4e6cc8...24f052a

```
* 24f052a (tag: v0.25.1) Cut a 0.25.1 point release.
| * f4e6cc8 (origin/k8s-v1.9.4) Cut a 0.25.1 point release.
|/  
* 9dcc3b6 Update Kubernetes to 1.9.4, on top of the v0.25.0 branch.
*   c8a2016 (tag: v0.25.0)
```

**Pull request closed without merging**